docs/man/renderd.conf.5

```diff
@@ -0,0 +1,303 @@
+.TH RENDERD.CONF 5 "November 30, 2023"
+.\" Please adjust this date whenever revising the manpage.
+
+.SH NAME
+renderd.conf \- Configuration file of rendering daemon for rendering OpenStreetMap tiles.
+
+.SH SYNOPSIS
+renderd.conf
+
+.SH DESCRIPTION
+This manual page documents briefly the
+.B renderd.conf
+configuration file.
+.PP
+.B renderd.conf
+is an INI configuration file for use with
+.B renderd(1).
+.PP
+The default location is
+.B '/etc/renderd.conf'
+(e.g.
+.B RENDERD_CONFIG)
+.PP
```

Comment:
I think the closing parens shouldn't be bolded:
```suggestion
.B RENDERD_CONFIG
)
.PP
```

The same thing present multiple times

Reply:
Yes, I tried that and it adds a tab after the bolded term. I will try and use `\fB`...`\fR`